sklearn/preprocessing/data.py

```diff
@@ -1698,14 +1698,11 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    n_values : 'auto', int or array of ints
-        Number of values per feature.
+    classes : 'auto', 2D array of ints or strings or both.
```

Comment:
Are you also suggesting to rename `n_values` to `values`?

I agree that it is confusing and I have to read the documentation again to figure out each feature should have categories that are in `range(n_values)` rather than `n_values` possible categories.

This should cause even more confusion now that we support categories that are also negative.